src/sentry/buffer/redis.py

```diff
@@ -19,6 +20,24 @@
 _local_buffers = None
 _local_buffers_lock = threading.Lock()
 
+logger = logging.getLogger(__name__)
+
+# Debounce our JSON validation a bit in order to not cause too much additional
+# load everywhere
+_last_validation_log = None
+
+
+def _validate_json_roundtrip(value, model):
+    global _last_validation_log
+
+    if _last_validation_log is None or _last_validation_log < time() - 10:
```

Comment:
If this only once every 10 seconds, will this actually be running often enough?

Reply:
i don't know. if this doesn't have any impact I'll follow up with reducing this interval

normally we'd do sample rates here but it's hard for me to reason about this when I have no clue which pods call buffer.incr at which rate. at the same time I can be sure one error every 10 seconds from each pod is probably fine